contrib/cryptomb/private_key_providers/source/cryptomb_private_key_provider.cc

```diff
@@ -522,7 +524,8 @@ CryptoMbPrivateKeyMethodProvider::CryptoMbPrivateKeyMethodProvider(
       stats_(generateCryptoMbStats("cryptomb", factory_context.statsScope())) {
 
   if (!ipp->mbxIsCryptoMbApplicable(0)) {
-    throw EnvoyException("Multi-buffer CPU instructions not available.");
+    ENVOY_LOG(warn, "Multi-buffer CPU instructions not available.");
```

Comment:
I personally think warn would be better because if this extension is unavailable means users have error configuration that should be corrected.

Reply:
Envoy may running on a machine without required hardware. After we add new private key provider, then the user can fill a list of providers which want to use. Then Envoy will choice the one available. In this case, the warn log may annoy the user.

Reply:
But with the change, the exist users couldn't get the configuration error info easily, and because the exception is removed, you can never get the info from exceptiom message. This improve the complexity of debugging.

If you don't want to annoy users of the new feature, we still have some other options:
1) keep the exception and catch it by the caller. The caller will decide to throw it again or ignore it based on the `fallback` setting.
2) propagate the error info by absl::Status or something. The caller could log/throw it or ignore it based on the `fallback` setting.

Reply:
After second thinking, I think the warn is fine here. I missed that, this is only log when we create the provider which happened only when sds update or envoy startup, so it is totally fine.